core/arch/arm/plat-stm32mp2/drivers/stm32mp25_syscfg.c

```diff
@@ -0,0 +1,42 @@
+// SPDX-License-Identifier: BSD-2-Clause
+/*
+ * Copyright (c) 2024, STMicroelectronics
+ */
+
+#include <drivers/stm32_shared_io.h>
+#include <initcall.h>
+#include <io.h>
+#include <mm/core_memprot.h>
+#include <stdbool.h>
+#include <stm32_sysconf.h>
+#include <stm32_util.h>
+#include <types_ext.h>
```

Comment:
Sorry again, I see few changes needed in the header files inclusions:
```patch
-#include <initcall.h>
-#include <stdbool.h>
-#include <stm32_util.h>
+#include <assert.h>
+#include <util.h>
```

`#include <stdbool.h>` should come in the next commit and is needed only in the header file. Could you remove it, for consistency.